docs/transactions/iti41.md

```diff
@@ -1,35 +1,229 @@
 # ITI-41
 
-DocumentEntry metadata are described in [documents](documents.md). Additional constraints are described here.
+The [ITI-41 (Provide and Register Document Set-b)](https://profiles.ihe.net/ITI/TF/Volume2/ITI-41.html) transaction allows a [content sender](https://profiles.ihe.net/ITI/TF/Volume2/ITI-41.html#3.41.2) to send documents to the eMedication service ([content receiver](https://profiles.ihe.net/ITI/TF/Volume2/ITI-41.html#3.41.2)). 
+
+Generic [DocumentEntry metadata](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.1.1) are described in the [documents section](documents.md). This page references additional constraints specific to ITI-41. See also IHE's documentation for [metadata in Document Sharing profiles](https://profiles.ihe.net/ITI/TF/Volume3/index.html).
+
+## Publishing documents
+### Document types
+Two types of documents might be published to the service :
+- [CH-EMED-EPR](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/) documents. These documents contain the eMedication data. See also the [corresponding page](../emed/index.md) in this guide.
```

Comment:
Only stable CH-EMED-EPR documents can be published, i.e. PML and PMLC documents are CH-EMED-EPR (on-demand) documents and cannot be provided by clients through ITI-41.